.travis.yml

```diff
@@ -12,7 +12,8 @@ branches:
 before_script:
 - npm install -g gulp
 - gulp
-script: npm test
+test:
+  - npm test
```

Comment:
This should remain `script: npm test`. `test` is not a valid configuration for Travis.